spring-integration-core/src/main/java/org/springframework/integration/config/EnableIntegrationManagement.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2014-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.config;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.context.annotation.Import;
+import org.springframework.integration.support.management.IntegrationManagement;
+import org.springframework.integration.support.management.IntegrationManagementConfigurer;
+
+/**
+ * Enables default configuring of management in Spring Integration components in an existing application.
+ *
+ * <p>The resulting {@link IntegrationManagementConfigurer}
+ * bean is defined under the name {@code integrationManagementConfigurer}.
+ *
+ * @author Gary Russell
+ * @since 4.0
```

Comment:
??? 
I see this class is completely new, so, `4.2` on the class is enough and would be good.